valora/quiz/tests/test_models.py

```diff
@@ -0,0 +1,96 @@
+from django.test import TestCase
+from ..models import Category, Question, Answer, Classification
+from django.contrib.auth.models import User, Group
+
+
+# ------------------------------------------------------------------------
+# To run
+# python manage.py test quiz
+# ------------------------------------------------------------------------
+
+
+class CategoryTest(TestCase):
+    """ Test for Category model """
+
+    @classmethod
+    def setUpTestData(cls):
+        # Set up non-modified objects used by all test methods
+        Category.objects.create(name='Category1')
+
+    def test_category_field_name(self):
+        print('test_category_field_name')
+        cat = Category.objects.get(id=1)
```

Comment:
@luigus Por mais que seja apenas um test e fique óbvio oque esta na variável `cat` é interessante você colocar nomes descritivos (isso em todo código que for fazer), pois no futuro em algum refactor isso vai lhe ajudar bastante, e também quando trabalhar em time isso acaba sendo muito útil. Toda vez que for 'codar' se lembre: Você tem que fazer código para outros programadores conseguirem ler também, por isso clean code é sempre bem-vindo!

Reply:
Você tem razão, acho que estou mal acostumado em trabalhar em um time muito pequeno, onde não temos code-review.
Porque este feedback é super importante para o crecimento profissional de um desenvolvedor.